src/extensions/service-vendors/bandwidth/messaging.js

```diff
@@ -178,19 +178,38 @@ export async function postMessageSend({
         ...changes
       }
     : {};
+  const organizationContact = {
```

Comment:
Previously, an `organization_contact` row wasn't being created/updated when a Bandwidth outbound message is sent. It is necessary to create/update an `organization_contact` row because the `sticky-sender` service manager (mandatory for Bandwidth) queries the `organization_contact` table to send texts/replies to the contact from the same Bandwidth phone number